### PR TITLE
Formatting improvements

### PR DESCRIPTION
**Pull request by bmds**
Also fixed an issue with the linter being run on production rather than dev

### CODE REVIEW

File: gulpTasks/scripts.js [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
+
 /* ============================================================ *\
     SCRIPTS JS / lint, concat and minify scripts
 \* ============================================================ */
 
-var gulpif = require('gulp-if');
-var jshint = require('gulp-jshint');
-var concat = require('gulp-concat');
-var uglify = require('gulp-uglify');
+// Grunt dependencies
```
Comment (furzeface) [inline gulpTasks/scripts.js]: Gulp* @tawashley :wink:

Comment (furzeface) [inline gulpTasks/scripts.js]: Sorry, @bmds :wink:

**Comment by tawashley** [comment]
@furzeface Happy?

**Comment by furzeface** [comment]
@tawashley ecstatic.

**Comment by bmds** [comment]
Ha ha, that's an amazing mistake

**Pull request merged**